### PR TITLE
avatar size

### PR DESCRIPTION
**Pull request by mhasilau**

### CODE REVIEW

File: src/api/api-handlers.js [hunk 1]
```diff
@@ -157,8 +158,9 @@ export const getUsers = () => {
 export const loadPhoto = async (event, avaName) => {
   const user = LocalStorageClass.getUserData();
   awaiting();
-
-  await firebase
+  console.log(event.target.files[0].size);
```
Comment (AliakseiKliashchou) [inline src/api/api-handlers.js:161]: log

File: src/api/api-handlers.js [hunk 2]
```diff
@@ -157,8 +158,9 @@ export const getUsers = () => {
...
+  if (event.target.files[0].size < 2000000) {
+    await firebase
     .storage()
     .ref(`img/${avaName}`)
     .put(event.target.files[0])
```
Comment (AliakseiKliashchou) [inline src/api/api-handlers.js:166]: what if size had been too small or wrong format?

File: src/api/api-handlers.js (hunk 1 above)
Comment (mhasilau) [reply src/api/api-handlers.js:161]: delete

File: src/api/api-handlers.js (hunk 2 above)
Comment (mhasilau) [reply src/api/api-handlers.js:166]: upload photo less 2MB

**Pull request merged**